frontend/src/app/(vendor)/layout.tsx

```diff
@@ -15,6 +16,7 @@ export default function VendorLayout({
 
   return <>
     <Navbar isVendorNavbar={true} />
+    <Toolbar />
```

Comment:
What will the toolbar add on top of the navbar?

Reply:
Turns out Toolbar spacer is one of the recommended solutions by Material UI. It's a spacer the same width as the AppBar. Cleaned this up by moving `Toolbar` to be part of `Navbar.tsx`, and removing `Toolbar` usage in VendorEditProfile.tsx and Layout.tsx